build_msvc/common.init.vcxproj

```diff
@@ -6,7 +6,12 @@
     <VCProjectVersion>16.0</VCProjectVersion>
     <VcpkgTriplet Condition="'$(Platform)'=='Win32'">x86-windows-static</VcpkgTriplet>
     <VcpkgTriplet Condition="'$(Platform)'=='x64'">x64-windows-static</VcpkgTriplet>
-</PropertyGroup>
+    <QtBaseDir>C:\Qt5.9.7_ssl_x64_static_vs2017</QtBaseDir>
```

Comment:
`<QtBaseDir Condition="'$(QtBaseDir)'==''">` this allow somebody to put Qt where he wants (and specify it in the `common.init.vcxproj.user` of https://github.com/bitcoin/bitcoin/pull/16309)

Reply:
@NicolasDorier 

See https://github.com/bitcoin/bitcoin/pull/22890/commits/5e42f2ad26d35950405480f3eea2ed73fdfba841